HTML-Assignment-3 (Design Layouts)/Design Layout Dashbaord.html

```diff
@@ -0,0 +1,199 @@
+<!DOCTYPE html>
```

Comment:
Does not meet the design.

Reply:
I don't quite understand the design element behind the comment.

I'll make sure to come up with this to you tomorrow.